reviews two/swagger.yaml

```diff
@@ -0,0 +1,879 @@
+      `shipping_address` | The mailing address to where the draft order will be shipped. | `string`
+      `billing_address` | The mailing address associated with the payment method. | `string`
+      `note` | The text of an optional note that a shop owner can attach to the draft order. | `string`
+      `email` | The email address used for sending notifications. |`string`
+      `currency` | The three letter code for the currency to be used for the payment. | `string`
+      `invoice_sent_at` | DateTime when the invoice was emailed to the customer by Shopify. | `dateTime`
+      `invoice_url` | The url to send to the customer so that they can complete the checkout.  When using `send_invoice`, this url is emailed to the customer. This field can be used so that an API client can use another method of communication to provide the url to the customer. | `string`
+      `line_item`[ ] | | array of `line_item` objects
+      `metafields`[ ] | | array of `metafield` objects
+      `shipping_line` | | `object`
+      `tags` | Tags are additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. Each individual tag is limited to 40 characters in length. | `string`
+      `tax_exempt` | Sets whether taxes are exempt for this draft order. If this value is `false`, Shopify will honor the `tax_exempt` value for each `line_item`. | `boolean`
+      `tax_lines` | Tax lines describing the sum of each type of tax line for line items and shipping line. | array of `tax_line` objects
+      `discount` | Order level discount. | `string`
+      `taxes_included` | Shop settings taxes are included in the price | `boolean`
+      `total_tax` | Total tax amount | `integer`
+      `completed_at` | Date at which an order was created and the draft order moved to “completed” status. | `DateTime`
+      `created_at` | By default, this auto-generated property is the date and time when the order was created in Shopify, in ISO 8601 format. If you are importing orders to the Shopify platform from another system, the writable `processed_at` property will override the `created_at` date. | `dateTime`
+      `updated_at` | | `dateTime`
+      `status` | String describing the state of the draft order. Possible values are `open`, `invoice sent`, or `completed`. | `string`
+      
+
+     
+
+  
+  x-draftorder: 
+    type: object
+    required:
+    - title
+    - rate
+    - price
+    properties:
+      title:
+        type: string
+        description: title
+      rate:
+        type: string
+        description: Rate for this tax
+      price:
+        type: string
+        description: Total
+        
+
+  
+schemes:
+  - https
+host: johns-apparel.myshopify.com
+
+securityDefinitions:
+  api_key:
+    type: apiKey
+    description: apikey security
+    name: johns-api-key
+    in: header
+
+paths:
+  /admin/draft_orders.json:
+    get:
+      description: Retrieve draft orders
```

Comment:
Sometimes you have periods and sometimes you don't. Can you please be consistent?